### ISSUE TITLE
[Bug] ROCm/Linux 环境下无法导入 hypir_node 与 hypir_model_manager 模块/Import Error for hypir_node and hypir_model_manager on ROCm/Linux Environment

### ISSUE THREAD

**Issue by lokvahkooravidya** [11dogzi/Comfyui-HYPIR#22]
安装 Comfyui-HYPIR 插件后，ComfyUI 日志持续报出“Error loading module hypir_node: No module named 'hypir_node'”。虽然基础功能似乎通过某种保底逻辑可以运行，但核心节点映射未能正确加载。这似乎是 __init__.py 中的路径解析问题或内部文件名不匹配导致的。虽然工作流可以跑完并生成图像，但我并不确定输出的图像质量是否达到了预期（例如，可能存在细节丢失或非预期的伪影）。由于启动日志显示模块导入失败，我怀疑尽管基础计算能够完成，但某些关键配置或预训练权重可能并未以最佳状态加载。希望开发者能确认目前的 ImportError 是否会影响内部参数的初始化。

After installing the Comfyui-HYPIR plugin, the ComfyUI log consistently reports "Error loading module hypir_node: No module named 'hypir_node'". Although the base functions seem to work via fallback logic, the core node mappings are not being loaded correctly. It appears to be a path resolution issue within the __init__.py or a mismatch in internal filenames.While the workflow completes and produces an image, I am concerned that the output quality may not be meeting expectations (e.g., potential loss of detail or unintended artifacts). Since the startup logs show module import failures, I suspect that some critical configurations or pre-trained weights might not be loading optimally, even if the basic computation finishes. I would appreciate it if the developers could confirm if the current ImportError affects the internal parameter initialization.

Environment / 环境信息

OS: Linux (Ubuntu 24.04/Similar)
GPU: AMD Radeon RX 9070 XT (gfx1201)
ROCm Version: 7.2.0
PyTorch Version: 2.9.1+rocm7.2.0
Python Version: 3.12.12 | packaged by Anaconda, Inc. | (main, Oct 21 2025, 20:16:04) [GCC 11.2.0]
ComfyUI Version: 0.10.0
Attention: Pytorch Attention
Backend Status: Eager (Enabled), Triton (Disabled), Cuda-Wrapper (Disabled)
torch: 2.9.1+rocm7.2.0.lw.git7e1940d4
triton: 3.5.1+rocm7.2.0.gita272dfa8

说明： 在此 RDNA 4 配置下，Triton 和 Cuda-Wrapper 后端目前处于禁用状态，强制使用 Eager 模式运行。
Note: The Triton and Cuda-Wrapper backends are currently disabled, force-using Eager mode on this RDNA 4 setup. 

Error Logs / 错误日志

Error loading module hypir_node: No module named 'hypir_node'
Error loading module hypir_model_manager: No module named 'hypir_model_manager'
/home/shirin/comfy/ComfyUI/custom_nodes/Comfyui-HYPIR/hypir_config.py:32: SyntaxWarning: invalid escape sequence '\m'

File Structure Observation / 文件结构观察

我注意到在 HYPIR 子目录下，文件名是 hypir_advanced_node.py 和 model_downloader.py，但入口文件似乎在寻找 hypir_node.py。

I noticed that inside the HYPIR subdirectory, the files are named hypir_advanced_node.py and model_downloader.py, but the entry point seems to be looking for hypir_node.py. 

AI-Generated Disclaimer / 自动生成声明

    本 Issue 报告由大语言模型（LLM）根据用户提供的系统诊断数据和日志自动抓取并生成。测试环境使用了最前沿的 AMD RDNA 4 (gfx1201) 硬件。请开发者在排查时优先参考附带的原始终端输出及目录树结构。

    This issue report was automatically generated and summarized by an LLM based on system diagnostics and logs provided by the user. The hardware environment is a cutting-edge AMD RDNA 4 (gfx1201) setup. Please refer primarily to the attached raw terminal outputs and directory tree for troubleshooting.
